solaris/tests/private_contract.rs

```diff
@@ -0,0 +1,46 @@
+extern crate ethabi;
+#[macro_use]
+extern crate ethabi_derive;
+#[macro_use]
+extern crate ethabi_contract;
+extern crate ethereum_types as types;
+extern crate rustc_hex;
+extern crate solaris;
+
+use rustc_hex::{FromHex, ToHex};
+use solaris::sol;
+
+fn main() {
+    solaris::main(include_bytes!("../res/PrivateContract_sol_PrivateContract.abi"));
+}
+
+use_contract!(private_contract, "PrivateContract", "res/PrivateContract_sol_PrivateContract.abi");
+
+fn setup() -> (solaris::evm::Evm, private_contract::PrivateContract) {
+    let contract = private_contract::PrivateContract::default();
+    let code = include_str!("../res/PrivateContract_sol_PrivateContract.bin");
+
+    // PrivateContract initialization arguments
+    let validators = sol::address(10).to_hex()
+        + sol::address(20).to_hex().as_str()
+        + sol::address(30).to_hex().as_str();
+    let init_code = sol::bytes32("Asdfg").to_hex();
+    let init_state = sol::bytes32("Lkjhg").to_hex();
+
+    let deploy_call = String::from(code)
```

Comment:
you can just use `contract.constructor()` to have `ethabi` encode that for you automatically.